src/theme/qml/ComboBox.qml

```diff
@@ -44,32 +44,64 @@ T.ComboBox {
     hoverEnabled: true
 
     delegate: MenuItem {
-        width: parent.width
+        width: parent.width - 5
         text: control.textRole ? (Array.isArray(control.model) ? modelData[control.textRole] : model[control.textRole]) : modelData
         font.weight: control.currentIndex === index ? Font.DemiBold : Font.Normal
         highlighted: control.highlightedIndex == index
         hoverEnabled: control.hoverEnabled
+        x: control.mirrored ? control.width - width + 2 : 2
     }
 
-    indicator: Shape {
-        implicitHeight: 10
-        implicitWidth: 10
-        x: control.mirrored ? control.leftPadding - 2 : control.width - width - control.rightPadding + 2
-        y: control.topPadding + control.availableHeight / 2
-
-        ShapePath {
-            fillColor: theme.textColor
-            strokeColor: theme.textColor
-            capStyle: ShapePath.FlatCap
-            joinStyle: ShapePath.MiterJoin
-
-            startX: -4
-            startY: -2
-            PathLine { x: 0; y: 2 }
-            PathLine { x: 4; y: -2 }
+    indicator: Item {
+        implicitHeight: theme.comboBoxMinimumHeight
+        implicitWidth: theme.comboBoxMinimumHeight
+        x: control.mirrored ? control.leftPadding - 2 : control.width - width
+        layer.enabled: true
+
+        Rectangle {
+            id: roundCorners
+            color: theme.getButtonBottomColor(false, false, control.hovered,control.pressed || control.checked || popup.visible) 
+            radius: theme.frameRadius
```

Comment:
You use `theme.frameRadius` everywhere, you should use value (variable name) that really represents what one would expect when reading the code. You have them all listed [here](https://github.com/FedoraQt/MediaWriter/blob/main/src/theme/theme.h).